build/images/ovs/Dockerfile.windows

```diff
@@ -0,0 +1,46 @@
+# Copyright 2023 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG NANOSERVER_VERSION
+
+FROM mcr.microsoft.com/windows/servercore:${NANOSERVER_VERSION} as windows-ovs
+ARG WIN_OVS_VERSION
+
+# Switch to a privileged user
+USER ContainerAdministrator
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+ENV WIN_OVS_VERSION ${WIN_OVS_VERSION}
+
+# Install VC
+RUN curl.exe -Lo C:\vc_redist.x64.exe https://aka.ms/vs/17/release/vc_redist.x64.exe; \
+    powershell -Command  "Start-Process -FilePath C:\vc_redist.x64.exe -ArgumentList ‘/install’,’/quiet’,’/norestart’ -Verb RunAs -Wait"
+
+# Download and extract OVS
+RUN curl.exe -fLo C:\ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip https://downloads.antrea.io/ovs/ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip;  \
+    powershell -Command Expand-Archive C:\ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip -DestinationPath C:\  ;  \
+    rm C:\ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip
+
+# Download and extract OpenSSL
+RUN curl.exe -Lo C:\openssl-1.0.2u-x64_86-win64.zip https://indy.fulgan.com/SSL/openssl-1.0.2u-x64_86-win64.zip; \
+    powershell -Command Expand-Archive C:\openssl-1.0.2u-x64_86-win64.zip -DestinationPath C:\  ;  \
+    rm C:\openssl-1.0.2u-x64_86-win64.zip
+
+RUN cp C:/libeay32.dll C:/openvswitch/usr/sbin/ ; \
+    cp C:/ssleay32.dll C:/openvswitch/usr/sbin/ ; \
+    mv C:/libeay32.dll C:/openvswitch/usr/bin/ ; \
+    mv C:/ssleay32.dll C:/openvswitch/usr/bin/ ; \
+    mkdir C:/openvswitch/var/log/openvswitch; \
```

Comment:
I am thinking if `C:/openvswitch/var/log/openvswitch` and `C:/openvswitch/var/run/openvswitch` are necessary in the image? Or we can create them in the runtime?

Reply:
@XinShuYang @rajnkamr ^^^

Reply:
@wenyingd - these folders are required to run ovs daemons later, in case if these folders does not exist, daemons will not get started, while it can be created during runtime, it might not add much benefit and we want to be certain that these folders exists !

Reply:
I have one concern, if we create the directory inside the containers, will the files found on the storage we mounted to the Pod. For example, we always mount path "/var/log/antrea" on the Node as a volume "host-var-log-antrea" to antrea-agent Pod on Linux, and antrea-ovs container could output the log files to path "/var/log/antrea/openvswitch".  In this way, when antctl supportbundle collects the bundle files, it could include both the logs of antrea-agent and antrea-ovs. It is the Linux implementation. On Windows, we would also output antrea-agent logs to host path "C:/var/log/antrea", then where shall we collect the ovs logs? If we also use the host path "C:/openvswitch/var/log/", it means this directory created inside the container is not consumed, then why we must have this step to create it inside the image? On the other hand, if we expect to output OVS logs to this path, it may introduce more work in antctl to collect logs, and the user must enter the OVS container to read it?

Reply:
@wenyingd , wrt  mountPath , /var/log/openvswitch  is automatically created inside the container if it doesn't already exist during mount as amount is done via yaml, the case when we want to start ovs process from inside container these directories are required to be created via dockerfile.
 Hence we don't need to create the /var/log/openvswitch directory inside the container via dockerfile now.

Reply:
@wenyingd , 
Another folder C:/openvswitch/var/run/openvswitch will be needed to be created inside container for ovs runtime data.

Reply:
That depends on if you use it in the Pod right?

Reply:
yes it is getting used, run time ovs data like pid files etc gets stored inside this folder